caf-audit-monkey/README.md

```diff
@@ -0,0 +1,145 @@
+# Audit Monkey
+
+## Description
+The Audit Monkey provides the functional ability to test the Audit Service including the auto-generated AuditLog, (both the direct and webservice variants) and the Audit WebService.  
+The Audit Monkey has the ability to send Audit Events both directly to Elasticsearch and via the Audit WebService.
+The Audit Monkey also has the ability to generate significant volumes of data as it can be run in both single-threaded and multi-threaded modes.  
+
+## Configuration
+
+### Modes
+The Audit Monkey can be run in a number of modes. Details of these modes are outline in the table below.  
+The three key modes cover:  
```

Comment:
Could be my fault in reading this but are these really modes are running options? I would have thought a mode would have comprised of a combination of all 3 CAF\_AUDIT\_MODE, CAF\_AUDIT\_MONKEY\_MODE and CAF\_AUDIT\_MONKEY\_NUM\_OF\_THREADS.